desktop/core/src/desktop/templates/global_js_constants.mako

```diff
@@ -114,6 +114,7 @@
   window.ENABLE_QUERY_SCHEDULING = '${ ENABLE_QUERY_SCHEDULING.get() }' === 'True';
   window.ENABLE_UNIFIED_ANALYTICS = '${ ENABLE_UNIFIED_ANALYTICS.get() }' === 'True';
   window.RAZ_IS_ENABLED = '${ RAZ.IS_ENABLED.get() }' === 'True';
+  window.ENABLE_NEW_FILE_UPLOADER = '${ ENABLE_NEW_FILE_UPLOADER.get() }' === 'True';  
```

Comment:
Why are we adding it here if we have already added it in the `/get_config` response?

Reply:
I was not able to use the window object in listdir_components.mako without adding the config in global_js_constants.mako.

Reply:
@bjornalm @JohanAhlen Do we need some extra steps or different process for using the /get_config values on the UI side in the mako files? Or I guess we can just import it directly like in backend?

Reply:
Thanks @bjornalm! @athithyaaselvam Maybe give a quick try with the above suggestion? Because in the long term we've to get rid of this file and only depend on /get_config response for configs in UI.

Reply:
Thanks for the suggestion. Made changes to use the /get_config response.